scripts/common/rook.sh

```diff
@@ -67,46 +67,63 @@ function remove_rook_ceph() {
     # make sure there aren't any PVs using rook before deleting it
     all_pv_drivers="$(kubectl get pv -o=jsonpath='{.items[*].spec.csi.driver}')"
     if echo "$all_pv_drivers" | grep "rook" &>/dev/null ; then
-        # do stuff
-        printf "%b" "$RED"
-        printf "ERROR: \n"
-        printf "There are still PVs using rook-ceph.\n"
-        printf "Remove these PVs before continuing.\n"
-        printf "%b" "$NC"
+        logFail "There are still PVs using rook-ceph."
+        logFail "Remove these PVs before continuing."
         exit 1
     fi
 
     # scale ekco to 0 replicas if it exists
     if kubernetes_resource_exists kurl deployment ekc-operator; then
         kubectl -n kurl scale deploy ekc-operator --replicas=0
         log "Waiting for ekco pods to be removed"
-        if ! spinner_until 120 ekco_pods_gone; then
-             logFail "Unable to scale down ekco operator"
-             return 1
+        if ! spinner_until 300 ekco_pods_gone; then
+             logWarn "Unable to scale down ekco operator"
+             logWarn "The script will continue to try remove Rook"
```

Comment:
It is unlike to occur but if we be unable to make ekco down seems that still possible to delete Rook. Am I right?
@laverya have we any reason to stop the operation here instead of try to move forward and remove rook?